src/cosmos-ledger.ts

```diff
@@ -74,7 +76,22 @@ export default class Ledger {
     // assume well connection if connected once
     if (this.cosmosApp) return this
 
-    let transport = await TransportU2F.create(timeout * 1000)
+    let transport
+    if (isWindows(this.platform)) {
+      if (!navigator.hid) {
+        throw new Error(
+          "Your browser doesn't have HID enabled. Please enable 'Experimental Web Platform features' here: chrome://flags / brave://flags"
```

Comment:
```suggestion
          "Your browser doesn't have HID enabled. Please enable 'Experimental Web Platform' features here: chrome://flags / brave://flags"
```

Reply:
can we make `chrome://flags` a link some how? 

also, what specific feature needs to be enabled? the list is very long and we can select the anchor tag for the specific flag that needs to be changed.

Reply:
the item was called like this ^^ designers...